### PR TITLE
NOISSUE - Update docs

### PR DESCRIPTION
**Pull request by mteodor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,32 +25,76 @@ make
 ```
 
 ## Usage
+```
+go get github.com/nats-io/gnatsd
+gnatsd
+MF_AGENT_BOOTSTRAP_ID=11:22:33:44 MF_AGENT_BOOTSTRAP_KEY=5341a625-53a5-4a25-be74-5b648243fcbc MF_AGENT_NATS_URL=nats://localhost:4223 MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap build/mainflux-agent
```
Comment (manuio) [inline README.md:31]: This looks specific to your setup. I would prefer.
```
MF_AGENT_BOOTSTRAP_ID=<agent_bootstrap_id> \
MF_AGENT_BOOTSTRAP_KEY=<agent_bootstrap_key> \
... 
``` 
btw, this is normal? `nats://localhost:4223`

File: README.md [hunk 2]
```diff
@@ -25,32 +25,76 @@ make
 ```
 
 ## Usage
+```
+go get github.com/nats-io/gnatsd
+gnatsd
```
Comment (manuio) [inline README.md:30]: Can you explain what this command is doing?

File: README.md (hunk 1 above)
Comment (drasko) [reply README.md:31]: @manuio `4223` is default NATS port, but I think this one is not needed, as service could use this default by itself.

Comment (manuio) [reply README.md:31]: @drasko default NATS port is 4222.

File: README.md [hunk 3]
```diff
@@ -25,10 +25,17 @@ make
...
+Get Nats server and start it
 ```
 go get github.com/nats-io/gnatsd
 gnatsd
-MF_AGENT_BOOTSTRAP_ID=11:22:33:44 MF_AGENT_BOOTSTRAP_KEY=5341a625-53a5-4a25-be74-5b648243fcbc MF_AGENT_NATS_URL=nats://localhost:4223 MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap build/mainflux-agent
+```
+start Agent
+```
+MF_AGENT_BOOTSTRAP_ID=<boostrap_id> \
+MF_AGENT_BOOTSTRAP_KEY=<boostrap_key> \
```
Comment (manuio) [inline README.md:36]: `bootstrap`

File: README.md [hunk 4]
```diff
@@ -25,10 +25,17 @@ make
...
 
 ## Usage
+Get Nats server and start it
 ```
 go get github.com/nats-io/gnatsd
 gnatsd
-MF_AGENT_BOOTSTRAP_ID=11:22:33:44 MF_AGENT_BOOTSTRAP_KEY=5341a625-53a5-4a25-be74-5b648243fcbc MF_AGENT_NATS_URL=nats://localhost:4223 MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap build/mainflux-agent
+```
+start Agent
+```
```
Comment (manuio) [inline README.md:34]:
```bash

File: README.md (hunk 2 above)
Comment (manuio) [reply README.md:31]: you can use ```bash

File: README.md [hunk 5]
```diff
@@ -25,32 +25,83 @@ make
...
+    password = "0c6fd027-79fc-4fca-be27-814f7e842cf9"
+    priv_key_path = ""
+    qos = 0
+    retain = false
+    skip_tls_ver = false
+    url = "84.201.171.65:1883"
+    username = "fc430ab3-6765-49b0-af70-8629fe5d3900"
+
+  [Agent.server]
+    nats_url = "localhost:4223"
```
Comment (manuio) [inline README.md:73]: `localhost:4222"`

File: README.md [hunk 6]
```diff
@@ -25,32 +25,83 @@ make
...
+  [Agent.log]
+    level = "info"
 
-[log]
-level = "info"
+  [Agent.mqtt]
+    ca_path = ""
+    cert_path = ""
+    mtls = false
+    password = "0c6fd027-79fc-4fca-be27-814f7e842cf9"
```
Comment (manuio) [inline README.md:64]: local setup?

File: README.md [hunk 7]
```diff
@@ -25,32 +25,83 @@ make
...
+    priv_key_path = ""
+    qos = 0
+    retain = false
+    skip_tls_ver = false
+    url = "84.201.171.65:1883"
```
Comment (manuio) [inline README.md:69]: locall setup?

File: README.md [hunk 8]
```diff
@@ -25,32 +25,83 @@ make
...
+    username = "fc430ab3-6765-49b0-af70-8629fe5d3900"
```
Comment (manuio) [inline README.md:70]: locall setup?

**Pull request merged**